Zoocha/climateweeknyc
# TCG-116: New block to make sponsors page body editable

https://zoocha.atlassian.net/browse/TCG-116

New block has been created with `hook_update_N` to allow admins to edit the text that was hardcoded in the template.

**IMPORTANT**: Needs to be deployed after TCG-126 or at the same time as it modifies a view imported in that ticket.

DB changes: `drush updb` to create the block.

Manual changes: 

1. Check the custom block ID in `/admin/structure/block/block-content`
2. Add a new Rendered entity: Custom block in `/admin/structure/views/view/sponsors/edit/page_1` 
view header.

Test notes:
1. Go to http://climateweeknycijws48rsvv.devcloud.acquia-sites.com/sponsors and check the text is displayed.
2. Go to http://climateweeknycijws48rsvv.devcloud.acquia-sites.com/admin/structure/block/block-content and edit the block text.
3. Check the changes are applied.
